PKGBUILD

```diff
@@ -0,0 +1,35 @@
+# Maintainer: hodasemi <michaelh.95 at t-online dot de>
+_pkgbase=XRLinuxDriver
+pkgname="${_pkgbase}"
+pkgver=0.1
+pkgrel=1
+pkgdesc="XR Linux Driver"
+arch=('x86_64')
+url="https://github.com/wheaney/XRLinuxDriver"
+license=('GPL-3.0')
+makedepends=('cmake' 'make')
+depends=('openssl' 'libevdev' 'libusb' 'json-c')
+conflicts=("${_pkgbase}")
+source=("git+${url}")
+md5sums=(SKIP)
+
+build() {
+    cd ${_pkgbase}
+
+    # init submpdules
+    git submodule update --init --recursive
+
+    # build xr driver
+    mkdir build/
+    cd build
+    cmake ..
+    make
+}
+
+package() {
+    # copy xr driver
```

Comment:
This is missing the scripts in `bin`

Reply:
Which part is required exactly? The driver runs without crashing on my system in the current configuration.

I still have struggles with breezy-desktop, thus I could not interact with it.

Reply:
All these scripts are necessary: https://github.com/wheaney/XRLinuxDriver/blob/main/bin/setup#L41-L58

Reply:
I changed how the systemd service is run. In your script you change placeholder with the users home directory. That can all be omitted if you run the script as a user unit (`systemctl --user start xreal-air-driver`). All that is necessary for it to copy the script into `systemd/user/` instead of `systemd/system/`. With `sed` I changed the relevant part that the resulting file looks like:

```
[Unit]
Description=XREAL Air user-space driver
After=network.target

[Service]
Type=simple
ExecStart=xrealAirLinuxDriver
Restart=on-failure

[Install]
WantedBy=multi-user.target
```

I installed the `xrealAirLinuxDriver` binary in the system `/usr/bin` which is by default in `$PATH` thus no path is necessary to use the binary here anymore. The uninstall script makes no sense in a system wide install because on remove it won't be executed, left are only some dot files as with any program you use. 

Regarding the google analytics part: Honestly I don't think it is the right place to use, but who am I to judge. Nevertheless if the installer ever comes to an official repository the script is run once every version and the resulting package is then distributed rendering the execution on build time quite useless, even statistically.

Reply:
It's already been my intention to move this to a user service but it can't be done without also adding udev rules for all supported devices, so IMO that's another job for another PR if it's not essential for this one.

I don't need the Google Analytics in package manager installs and you're right that uninstall is unnecessary, I was just referring to the xreal_driver_config and verify scripts.

Reply:
The problem is that you can't say which user to add in the service file at build time and at installation time the package is fixed.

Reply:
Making the udev files quite essentially to make this MR work.

Reply:
I added the config script, but the verify script doesn't make much sense, since the executable is in another location the uninstall script is omitted.